packages/cli/src/user-management/email/templates/_common.mjml

```diff
@@ -0,0 +1,23 @@
+<mj-head>
+	<mj-attributes>
+		<mj-all font-family="Helvetica, Open Sans, system-ui, sans-serif"></mj-all>
+		<mj-text
+			font-weight="400"
+			font-size="16px"
+			color="#444444"
+			line-height="24px"
+			padding="10px 0 0 0"
+			align="center"
+		></mj-text>
+		<mj-button
+			background-color="#ff6d5a"
+			color="#ffffff"
+			font-size="18px"
+			align="center"
+			padding-top="20px"
+			line-height="150%"
+			border-radius="4px"
+		></mj-button>
```

Comment:
We might want these to match design system values.

Reply:
I've updated the colors and font-family to match the values in design-system.